ads/google/doubleclick.js

```diff
@@ -265,3 +247,54 @@ function doubleClickWithGlade(global, data, gladeExperiment) {
 function getCorrelator(global) {
   return makeCorrelator(global.context.clientId, global.context.pageViewId);
 }
+
+function centerAd() {
+  const container = global.document.querySelector('#c');
+  setStyles(dev().assertElement(container), {
```

Comment:
Done. (I just refactored this isn't a separate method and left the original code as is).